src/web/components/ParticipantRequests/ParticipantRequestItem.tsx

```diff
@@ -48,7 +48,7 @@ export function ParticipantRequestItem({
         <div className='participant-name'>{participant.name}</div>
       </td>
       <td>
-        <div className='participant-types'>{getParticipantTypes(participant.types)}</div>
+        <div className='participant-request-types'>{getParticipantTypes(participant.types)}</div>
```

Comment:
There might be some transition period between the existing names and external names - perhaps it would be handy to have a tooltip that shows the roleName? It may not strictly be necessary, so worth checking with the team.

Reply:
I really like that idea actually, since I wasn't really sure the best way to go about it anyway. I'll put that in there.

Reply:
Have added this in:
![chrome_rbJo6o15IE](https://github.com/IABTechLab/uid2-self-serve-portal/assets/152250444/e436d216-4557-49ac-9da5-8eb3fb935194)